### ISSUE TITLE
l3doc and fancyvrb

### ISSUE THREAD

**Issue by jlaurens** [latex3/latex3#1056]
l3doc should not use `\fvset` at the top level because it overrides the default settings of `fancyhdr`.
Instead, `fancyvrb` allows to create custom verbatim environments.

**Comment by josephwright** [comment]
This is true in principle but at present we are not planning to change - the class is rather a hack at present. Leaving this open for when a refactor takes place.

**Comment by josephwright** [comment]
To be tackled within the scope of https://github.com/latex3/latex3/issues/1136
